pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -1417,7 +1421,7 @@ func createRequestMappings(claim *resourceapi.ResourceClaim, pod *v1.Pod) []v1.C
 			for _, devReqName := range deviceReqNames {
 				// During filter phase, device request name is set to be
 				// container name index "-" extended resource name index
-				if fmt.Sprintf("container-%d-request-%d", i, ridx) == devReqName {
+				if strings.HasPrefix(devReqName, fmt.Sprintf("container-%d-request-%d", i, ridx)) {
```

Comment:
pre-existing, but fmt.Sprintf inside a loop for comparisons tends to be expensive, allocation-wise ... something to note and check for when doing benchmarking on this for efficiency

Reply:
ack, is there better alternative?